include/linker/common-rom.ld

```diff
@@ -172,6 +172,10 @@
 	} GROUP_LINK_IN(ROMABLE_REGION)
 #endif /* CONFIG_PCIE */
 
+#if defined(CONFIG_NEWLIB_LIBC_HEAP_LISTENER)
+	ITERABLE_SECTION_ROM(z_libc_heap_listener, 4)
```

Comment:
Please also add similar entry after the `CONFIG_PCIE` here:
https://github.com/zephyrproject-rtos/zephyr/blob/0d4f685dbdb521a35abbb2ad764b775eb3ecdc66/cmake/linker_script/common/common-rom.cmake#L154-L158

something like:
```
if(CONFIG_NEWLIB_LIBC_HEAP_LISTENER)
  zephyr_iterable_section(NAME z_libc_heap_listener KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN 4)
endif()
```

Reply:
Done